### ISSUE TITLE
can't configure customErrorResponse in ex.reactApp resource

### ISSUE THREAD

**Issue by ainvoner** [winglang/wing#4931]
### I tried this:

configure `customErrorResponse` for my react app when using ex.reactApp resource.

### This happened:

there is no way to configure it

### I expected this:

**ex.reactApp to be fully customizable**. There are many variations of a react app out there. we are covering only the basic form.
if we will have the resource more customizable is will help avoid non intuitive escape hatches.
In this case we needed to handle the error response differently than the default.

### Is there a workaround?

```js
if util.env("WING_TARGET") == "tf-aws" {
  let distributionNode = unsafeCast(std.Node.of(website).children.at(0).node.findChild("Distribution"));
  distributionNode.addOverride("custom_error_response", [
    {
      error_code: 403,
      response_code: 200,
      response_page_path: "/apps/index.html",
    },
  ]);
}
```

### Component

SDK

### Wing Version

latest

### Platform(s)

MacOS

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

**Comment by staycoolcall911** [comment]
@tsuf239 - was this resolved in #5307?

**Comment by tsuf239** [comment]
yes, I think so
